server/src/services/series.js

```diff
@@ -37,6 +39,86 @@ const aggregateSeries = async (seriesIds) => {
     ]);
 };
 
+const filterSeries = async ({ name, status, genre }, pageNumber, pageLimit) => {
```

Comment:
When name & status & genre are `undefined`, all series are returned (with pagination).
When name is `""`, status & genre are `undefined`, all series are returned (with pagination).
Is it expected? Because the Client will basically remove all checkboxes and will expect to `[]`.
Also: if he intentionally unticks all checkboxes for status, or for genre, I expect to get `[]` as well.

Reply:
I believe that without filters we should see all series